packages/wasm-dpp/test/unit/dataContract/stateTransition/DataContractCreateTransition/validation/state/validateDataContractCreateTransitionStateFactory.spec.js

```diff
@@ -31,55 +35,54 @@ describe('validateDataContractCreateTransitionStateFactory', () => {
 
     stateTransition.setExecutionContext(executionContext);
 
-    validateDataContractCreateTransitionState = validateDataContractCreateTransitionStateFactory(
-      stateRepositoryMock,
-    );
+    dataContractFetched = false;
+
+    const stateRepositoryLike = {
+      fetchDataContract: () => {
+        dataContractFetched = true;
+      },
+    };
+
+    factory = (t) => validateDataContractCreateTransitionState(stateRepositoryLike, t);
   });
 
-  it('should return invalid result if Data Contract with specified contractId is already exist', async () => {
-    stateRepositoryMock.fetchDataContract.resolves(dataContract);
+  // TODO wait for error types
+  // it('should return invalid result if Data Contract is already exist', async () => {
+  //   stateRepositoryMock.fetchDataContract.resolves(dataContract);
 
-    const result = await validateDataContractCreateTransitionState(stateTransition);
+  //   const result = await validateDataContractCreateTransitionState(stateTransition);
```

Comment:
Okay, so what's the plan here? Consensus errors are now in the v24 branch, so technically it's possible to check this. Do you want to do it here or in a separate PR?